src/citrine/informatics/predictor_evaluation_metrics.py

```diff
@@ -0,0 +1,189 @@
+        return "ndme"
+
+    def __str__(self):
+        return "NDME"
+
+
+class StandardRMSE(Serializable["StandardRMSE"], PredictorEvaluationMetric):
+    """[ALPHA] Standardized root-mean-square error."""
+
+    typ = properties.String("type", default="StandardRMSE", deserializable=False)
+
+    def __repr__(self):
+        return "standardized_rmse"
+
+    def __str__(self):
+        return "Standardized RMSE"
+
+
+class PVA(Serializable["PVA"], PredictorEvaluationMetric):
+    """[ALPHA] Predicted vs. actual data.
+
+    Results are returned as a flattened list, where each item represents
+    predicted vs. actual data for a single point.
+    """
+
+    typ = properties.String("type", default="PVA", deserializable=False)
+
+    def __repr__(self):
+        return "predicted_vs_actual"
+
+    def __str__(self):
+        return "Predicted vs Actual"
+
+
+class F1(Serializable["F1"], PredictorEvaluationMetric):
+    """[ALPHA] Support-weighted F1 score."""
+
+    typ = properties.String("type", default="F1", deserializable=False)
+
+    def __repr__(self):
+        return "f1"
+
+    def __str__(self):
+        return "F1 Score"
+
+
+class AreaUnderROC(Serializable["AreaUnderROC"], PredictorEvaluationMetric):
+    """[ALPHA] Area under the receiver operating characteristic (ROC) curve."""
+
+    typ = properties.String("type", default="AreaUnderROC", deserializable=False)
+
+    def __repr__(self):
+        return "area_under_roc"
+
+    def __str__(self):
+        return "Area Under the ROC"
+
+
+class CoverageProbability(Serializable["CoverageProbability"], PredictorEvaluationMetric):
+    """[ALPHA] Percentage of observations that fall within a given confidence interval.
+
+    The coverage level can be specified to 3 digits, e.g. 0.123 but not 0.1234.
+
+    Parameters
+    ----------
+    coverage_level: Union[str, float]
+        Confidence-interval coverage level.
+        The coverage level must both be between 0 and 1.0 (non-inclusive) will be rounded
+        to 3 significant figures.
+
+    """
+
+    _level_str = properties.String("coverage_level")
+    typ = properties.String("type", default="CoverageProbability", deserializable=False)
+
+    def __init__(self, coverage_level: Union[str, float]):
+        if isinstance(coverage_level, str):
+            try:
+                _level_float = float(coverage_level)
+            except ValueError:
+                raise ValueError(
+                    "Invalid coverage level string '{requested_level}'. "
+                    "Coverage level must represent a floating point number between "
+                    "0 and 1 (non-inclusive).".format(
+                        requested_level=coverage_level
+                    ))
+            if _level_float >= 1.0 or _level_float <= 0.0:
+                raise ValueError("Coverage level must be between 0 and 1 (non-inclusive).")
+            _level_float = round(_level_float, 3)
+            if len(coverage_level) > 5:
```

Comment:
This is very minor, but there maybe strings that have more than 3 sig figs that don't trigger this warning, for example: `".1111"`. Perhaps we can compare the pre rounded value to the the post rounded value instead?

Reply:
Adding a test for the warning too.